e2e-tests/insurance/cypress/e2e/journeys/no-access-to-application/no-access-to-application-abandoned.spec.js

```diff
@@ -0,0 +1,41 @@
+import { INSURANCE_ROUTES } from '../../../../../constants/routes/insurance';
+
+const {
+  ROOT,
+  ALL_SECTIONS,
+  NO_ACCESS_TO_APPLICATION,
+} = INSURANCE_ROUTES;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - no access to application page - Abandoned application', () => {
```

Comment:
Do we have user story type copy to add add the end here?

Reply:
done